### DISCUSSION TITLE
Add support for xxh3-64

### DISCUSSION THREAD

**Discussion by vivadavid** [doublecmd/doublecmd#1820] [Suggestions]
Hi,

I wanted to suggest adding support for the xxh3-64 checksum. I know that there are many types of hashes to consider, but as far as I know, this particular algorithm is quite fast and fit for those cases where you just want to make sure that your files aren't corrupted.

Thanks in advance.

**Comment by alexx2000** [comment]
#1779

**Reply by vivadavid** [reply]
> #1779

Thanks! I searched for "xxh3-64" and found nothing: that's why I opened this thread. I should have also tried the more general "xxHash".
